src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigWatcher.java

```diff
@@ -94,6 +95,7 @@ public void doRun() {
         for(String namespace:namespaces) {
           try {
             logger.fine("listing BuildConfigs resources");
+            getOpenShiftClient().getConfiguration().setOauthToken(CredentialsUtils.getCurrentToken());
```

Comment:
I've made a method getAuthenticatedOpenShiftClient() to replace the uses of getOpenShiftClient() that need authentication, will push after test